skopt/tests/test_searchcv.py

```diff
@@ -0,0 +1,80 @@
+"""This script contains set of functions that test parallel optimization with
+skopt, where constant liar parallelization strategy is used.
+"""
+
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_raises
+from sklearn.utils.testing import assert_greater
+
+from skopt.space import Real
+from skopt import Optimizer
+from skopt.benchmarks import branin
+import skopt.learning as sol
+
+from scipy.spatial.distance import pdist
+import pytest
+
+# Extract available surrogates, so that new ones are used automatically
+available_surrogates = [
+    getattr(sol, name) for name in sol.__all__
```

Comment:
I would rather  be explicit and write down the full list of surrogates, instead of relying on some black magic that could easily break down as we add/update things.